src/main/webapp/app/course/manage/course-management.component.html

```diff
@@ -103,6 +106,8 @@ <h4 id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses
                         </td>
                         <td>{{ course.startDate | artemisDate }}</td>
                         <td>{{ course.endDate | artemisDate }}</td>
+                        <td *ngIf="course.testCourse">{{ 'artemisApp.course.testCourse.title' | translate }}</td>
+                        <td *ngIf="!course.testCourse">{{ course.semester }}</td>
```

Comment:
this should not be mutually exclusive. We should be able to set test and semester and then we should also display both